docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,297 @@
+    INSERT INTO `weight` VALUES('DC1','DC1-2',11); 
+    INSERT INTO `weight` VALUES('DC2','DC2-1',9);
+    INSERT INTO `weight` VALUES('DC2','DC2-2',12);
+    INSERT INTO `weight` VALUES('DC3','DC3-1',11);
+    ```
+
+    The node in the cluster with the highest value is the first to be selected.
+
+11. Provision the remote clusters and start replication. 
+
+    On one of the DC1 nodes, for example DC1-1, perform a `mysqldump` as root or using sudo:
+
+    ```bash
+    [root@DC1-1 ~]# mysqldump -u root -p --source-data=2 --single-transaction -R -A -E > dump.sql
+    ```  
+
+12. Compress the file if it's too large. You can use [Percona XtraBackup to compress the data](https://docs.percona.com/percona-xtrabackup/8.0/backup_scenarios/compressed_backup.html). Copy the backup file to one node in each remote cluster, for example to DC2-1 and DC3-1. 
+
+    To restore the dump, run the following commands as root or using sudo:
+
+    ```bash
+    [root@DC2-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+    and
+
+    ```bash
+    [root@DC3-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+13. Configure the replication. Set up the first replication links manually. 
+
+    For example, on DC2-1 run:
+
+    ```bash
+    mysql> change replication source to source_host='WAN IP of DC1-1', source_user='repl', source_password='replpass', SOURCE_AUTO_POSITION = 1 FOR CHANNEL 'DC2-DC1';
+    mysql> start replica FOR CHANNEL 'DC2-DC1';
+    ```
+    
+    on DC3-1 run: 
+
+    ```bash
+    mysql> change replication source to source_host='WAN IP of DC1-1', source_user='repl', source_password='replpass', SOURCE_AUTO_POSITION = 1 FOR CHANNEL 'DC3-DC1';
+    mysql> start replica FOR CHANNEL 'DC3-DC1';
+    ```
+
+    For the other direction, use DC1-1 for both:
+
+    ```bash
+    mysql> change replication source to source_host='WAN IP of DC2-1', source_user='repl', source_password='replpass', SOURCE_AUTO_POSITION = 1 FOR CHANNEL 'DC1-DC2';
+    mysql> start replica FOR CHANNEL 'DC1-DC2';
+    mysql> change replication source to source_host='WAN IP of DC3-1', source_user='repl', source_password='replpass', SOURCE_AUTO_POSITION = 1 FOR CHANNEL 'DC1-DC3';
+    mysql> start replica FOR CHANNEL 'DC1-DC3';
+    ```
+
+    Now, all the clusters are linked in a source to source way. 
+
+14. Pull in the `replication_manager.sh` script. On each node, perform the following steps:
+
+    ```bash
+    $ cd /usr/local/bin
+    $ wget https://github.com/y-trudeau/Mysql-tools/raw/master/PXC/replication_manager.sh
+    $ chmod u+x replication_manager.sh
+    ```
+
+    When executed for the first time, the replication manager detects the current replication links and inserts rows in to the `percona.replication table`. In order to avoid issues, start with the replica nodes. The mysql credentials should be specified in the `/root/.my.cnf` file. On these nodes (DC1-1, DC2-1 and DC3-1), execute the script manually:
+    
+    ```bash  
+    $ ./usr/local/bin/replication_manager.sh
```

Comment:
The script location is `/usr/bin ` , so `/usr/bin/replication_manager.sh` or simply run command `replication_manager.sh`